components/ide-service/pkg/server/server.go

```diff
@@ -254,6 +254,13 @@ func (s *IDEServiceServer) ResolveWorkspaceConfig(ctx context.Context, req *api.
 		IdeImageLayers:  defaultIde.ImageLayers,
 	}
 
+	if os.Getenv("CONFIGCAT_SDK_KEY") != "" {
+		resp.Envvars = append(resp.Envvars, &api.EnvironmentVariable{
+			Name:  "SUPERVISOR_CONFIGCAT_ENABLED",
```

Comment:
Why the different env variable? Can we not use the standard CONFIGCAT_* for this?

Reply:
fyi, system reserved only from `GITPOD_`. ws-manager does not care about `SUPERVISOR` prefix. if we don't want a user to control it then name it `GITPOD_CONFIGCAT_ENABLED`